src/Frontend/Modules/Search/Ajax/Livesuggest.php

```diff
@@ -221,7 +221,7 @@ private function parsePagination(): void
             if ($useQuestionMark) {
                 $url = $this->pagination['url'] . '?page=' . ($this->pagination['requested_page'] - 1);
             } else {
-                $url = $this->pagination['url'] . '&amp;page=' . ($this->pagination['requested_page'] - 1);
```

Comment:
what bug does this solve?

Reply:
Go to http://demo.fork-cms.com/nl/zoeken and search fe on lor, click on page 2, you will see that the paging (generated by ajax) will not work because of the &amp; in the url